examples/justfile

```diff
@@ -6,11 +6,82 @@
 PID_DIR := "/tmp/bulletin-pids"
 
 # Default recipe - run the complete PAPI workflow test
-default: run-authorize-and-store
+default: (run-authorize-and-store "smoldot" "bulletin-polkadot-runtime")
 
-# Build the bulletin chain node in release mode
-build:
-    cargo build --release -p polkadot-bulletin-chain
+# Setup prerequisites for Westend runtime (polkadot and polkadot-parachain binaries)
+# This recipe clones polkadot-sdk, builds required binaries, and copies them to ~/local_bridge_testing/bin
+setup-westend-prerequisites:
+    #!/usr/bin/env bash
+    set -e
+    
+    BIN_DIR=~/local_bridge_testing/bin
+    POLKADOT_SDK_DIR=~/local_bridge_testing/polkadot-sdk
+    
+    echo "🔧 Setting up Westend runtime prerequisites..."
+    echo "   Target directory: $BIN_DIR"
+    
+    # Create bin directory
+    mkdir -p $BIN_DIR
+    
+    # Clone polkadot-sdk if it doesn't exist
+    if [ ! -d "$POLKADOT_SDK_DIR" ]; then
+        echo "   Cloning polkadot-sdk repository..."
+        git clone https://github.com/paritytech/polkadot-sdk.git $POLKADOT_SDK_DIR
+    else
+        echo "   polkadot-sdk already exists at $POLKADOT_SDK_DIR"
+    fi
+    
+    cd $POLKADOT_SDK_DIR
+    
+    # Check out the required branch
+    echo "   Checking out branch bko-bulletin-para-support..."
+    git fetch origin
+    # TODO: later, before merging, switch to the `bko-bulletin-support` https://github.com/paritytech/polkadot-sdk/pull/10662 (not now)
+    git reset --hard origin/bko-bulletin-para-support
+    
+    # Build polkadot binary
+    echo "   Building polkadot binary (this may take a while)..."
+    DYLD_FALLBACK_LIBRARY_PATH=/Library/Developer/CommandLineTools/usr/lib cargo build -p polkadot -r
```

Comment:
if `DYLD_FALLBACK_LIBRARY_PATH=/Library/Developer/CommandLineTools/usr/lib` is really needed (ok, for macos local), let's do this just once at the beginning like:

```
if [[ "$OSTYPE" == "darwin"* ]]; then
      export DYLD_FALLBACK_LIBRARY_PATH=/Library/Developer/CommandLineTools/usr/lib
fi
```